samples/cpp/benchmark_app/main.cpp

```diff
@@ -376,8 +375,11 @@ int main(int argc, char* argv[]) {
             auto& device_config = config[device];
 
             // high-level performance modes
-            auto ov_perf_hint = get_performance_hint(device, core);
-            device_config.emplace(ov::hint::performance_mode(ov_perf_hint));
+            if (!device_config.count(ov::hint::performance_mode.name())) {
```

Comment:
the config passed from CMD line has higher priority than the config loading from json file, which means overwrite will happen here if same key existed, rgiht?